doc/en/components/calendar.md

```diff
@@ -0,0 +1,384 @@
+---
+title: $Platform$ Calendar | Calendar | Infragistics
+_description: Create an intuitive calendar for applications to display date information, and users to input date information with $ProductName$ Calendar component.
+_keywords: $Platform$ Calendar, $ProductName$, Infragistics
+---
+
+# $Platform$ Calendar Overview
+The $ProductName$ Calendar component provides an easy and intuitive ways to display date information. Users can choose from three different selection modes - single selection, multiple selection or range selection.
+
+## $Platform$ Calendar Example
+<div class="divider--half"></div>
+
+<code-view style="height: 480px"
+           data-demos-base-url="{environment:dvDemosBaseUrl}"
+           iframe-src="{environment:dvDemosBaseUrl}/scheduling/calendar-overview"
+           alt="$Platform$ Calendar Example"
+           github-src="scheduling/calendar/overview">
+</code-view>
+
+## Usage
+
+To get started with the Calendar component, first you need to install $ProductName$ package by typing the following command:
+
+<pre style="background:#141414;color:white;display:inline-block;padding:16x;margin-top:10px;font-family:'Consolas';border-radius:5px;width:100%">
+npm install {PackageComponents}
+</pre>
+
+The next step is to import the `IgcCalendarComponent`:
+
+```ts
+import { IgcCalendarComponent } from 'igniteui-webcomponents/src/components/calendar/calendar';
+```
+
+### Single Selection Calendar
+
+Instantiating the `IgcCalendarComponent` is as easy as placing it in the html. This will display the current month in the single selection mode.

```

Comment:
```suggestion
Instantiating the `IgcCalendarComponent` is as easy as placing it in the html. This will display the current month in single selection mode.

```